pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
```

Comment:
please keep the space between the import groups
it differ between the kubevirt ones and the global ones

Reply:
Done. Can you please take a look?